README.md

```diff
@@ -68,6 +68,32 @@ parameters:
         container_xml_path: var/cache/dev/App_KernelDevDebugContainer.xml
 ```
 
+The XML file should exist prior to running PHPStan.
+
+If you want this file to be created automatically and resolved based on `APP_ENV` you can use this:
+
+```yaml
+parameters:
+    symfony:
+        container_xml_path: tests/container-loader.php
```

Comment:
This is a bit too magic for my taste. It's called `container_xml_path`. There should be a different config key for this scenario.

Reply:
If both config keys are filled  with a value, the extension should throw an exception so that the user knows they should provide either XML or PHP, but not both.

Reply:
Hmm, not sure if that makes it more clear. 

The `container_xml_path` is just points to a PHP file that returns the XML. 

How would you call it?

Reply:
Maybe we should make it more general and instead have a file that returns the whole container (which we can turn into the XML ourselves for our own purposes).

Reply:
And then it could be called `container_loader`

Reply:
Before doing it like this, I tried to create a new `ServiceContainerServiceMapFactory` and `ServiceContainerParameterMapFactory` that would actually load the real container via a separate `container_loader` file.

The problem is, the container is compiled, and does not contain the data we need.